dnsflow_read.py

```diff
@@ -255,8 +309,8 @@ def process_pkt(dl_type, ts, buf, stats_only=False):
 
             try:
                 name_set = struct.unpack(fmt,
-                        dnsflow_pkt[cp:cp + struct.calcsize(fmt)])[0]
-            except struct.error, e:
+                        dnsflow_pkt[cp:cp + struct.calcsize(fmt)])[0].decode("utf-8")
```

Comment:
This was the only real python 2->3 issue that 2to3 didn't do automatically. Unfortunately, this means the tool will no longer work with python2. If it is a hard requirement for it to work with python2 as well, I'm sure I can figure out a (probably hacky) way to make this happen...